rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -170,6 +176,8 @@ synchronized void handleMessage(DiscoveryEvent event) {
         //have a networkId in the message yet, then just let them through, for now.
         if (event.getMessage().getNetworkId().isPresent() &&
                 event.getMessage().getNetworkId().getAsInt() != this.networkId) {
+            logger.debug("handleMessage - Message ignored because network id [{}] is different from [{}]",
```

Comment:
```suggestion
            logger.warn("handleMessage - Message ignored because remote peer's network id: [{}] is different from local network id: [{}]",
```

Reply:
Done